FrontEnd/src/utils/validateRnokpp.js

```diff
@@ -0,0 +1,23 @@
+const RNOKPP_PATTRN = /^\d{10}$/;
+
+function validateRnokpp(rnokpp) {
+    if (RNOKPP_PATTRN.test(rnokpp)) {
+        const valueForValidation = Array.from(rnokpp).map(Number);
+        const weightCoeffBase = [-1, 5, 7, 9, 4, 6, 10, 5, 7];
+        let sum = 0;
+        for (let i = 0; i < 9; i++) {
+            sum += weightCoeffBase[i] * valueForValidation[i];
+        }
```

Comment:
Could you provide a comment on this?

Reply:
Comments explaining the validation have been added.